tests/queries/0_stateless/02982_changeDate.sql

```diff
@@ -0,0 +1,42 @@
+SELECT changeYear(toDate('2000-01-01'), 2001);
```

Comment:
The tests are good starting point but we need coverage of all edge cases:
- an input argument count != 2
- non-integer (e.g. string) arguments
- second arguments with different widths (e.g. 8/16/... bit, signed/unsigned)
- second arguments that cause the result to under or overflow the bounds of the data type.

^^ for error cases, you can use `serverError` (please grep it in tests/) to check for expected error codes)